### PR TITLE
Reformatted Test Reporting APIs' blueprints according to current standards

### PR DESCRIPTION
**Pull request by honzajavorek**
Also fixes 'array member definition of type object contains value' warnings.

**This Pull Request is tied to one internal Apiary Pull Request. Do not merge this before you know the whole context of this change.**

### CODE REVIEW

**Comment by w-vi** [comment]
You haven't changed any code, do we really need to wait for the CI test?

**Comment by honzajavorek** [comment]
@w-vi this is a bit stale - I should have assigned In Progress label to it. It's bound to some changes in Apiary and I wanted someone to at least read the changes. The CI statuses are pending, because they are now required in the repository, but this PR is from times before they were present, so they will never actually start. In other words, the PR needs rebase against master to pass.

File: ApiaryReportingApi.apib [hunk 1]
```diff
@@ -8,51 +8,53 @@ HOST: http://api.apiary.io/
...
+    - status (string) - Test run result
     - Values
-      - `success`
-      - `fail`
-      - `running`
-  - result (object) Result report
+        - `success`
+        - `fail`
+        - `running`
+    - result (object) Result reportt
```
Comment (miiila) [inline ApiaryReportingApi.apib]: Typo?

File: ApiaryReportingApi.apib [hunk 2]
```diff
@@ -8,51 +8,53 @@ HOST: http://api.apiary.io/
...
-  - endedAt (number) - Unix timestamp when test run ended in UTC
-  - createdAt (string) - Date in ISO format
-  - status (string) - Test run result
+    - _id: `507f1f77bcf86cd799439011` (string, optional) - Unique object id. DO NOT USE IT WHEN CREATING! Used in post body only for purpose of testing this blueprint.
+    - endpoint: `http://localhost:3000` (string, required) - protocol with hostname and optional PORT in URL format, root for all transactions in this Test Run
+    - reportUrl: `https://absolutely.fancy.url/wich-can-change/some/id` (string, required) - Report UI URL
+    - blueprints (array, required) - Array of blueprints used for the test run
+        - (object)
+            - filename (string, required) - file name
+            - raw (string, required) - raw bleuprint content in Markdown
```
Comment (miiila) [inline ApiaryReportingApi.apib]: typo: `bleuprint` => `blueprint`

File: ApiaryReportingApi.apib [hunk 3]
```diff
@@ -64,406 +66,365 @@ HOST: http://api.apiary.io/
...
-  - resultData (object) - Data from step execution
-    - request  - [Real HTTP Request](https://www.relishapp.com/apiary/gavel/v/0-1/docs/data-model#http-request)
-    - realResponse - [Real HTTP response](https://www.relishapp.com/apiary/gavel/v/0-1/docs/data-model#http-response)
-    - expected Response - [Expected HTTP Response](https://www.relishapp.com/apiary/gavel/v/0-1/docs/data-model#expected-http-response)
-    - result - [Validation result output](https://www.relishapp.com/apiary/gavel/v/0-1/docs/data-validators-and-output-format#validators-output-format)
-  - stepType (string, `exampleNameTransactionName`) - Type of the step, should be HTTP or Cucumber for instance
+    - _id (string) - Unique object id. DO NOT USE IT WHEN CREATING! Used in post body only for purpose of testing this blueprint.
+    - testRunId (string) - Reference to the parent test run identificator
+    - origin (object) - Origin path (position, location) of the step in the blueprint AST
+        - filename (string) .. File name of original blueprint
```
Comment (miiila) [inline ApiaryReportingApi.apib]: I'd take `..` off and put `-` instead.

File: ApiaryReportingApi.apib [hunk 4]
```diff
@@ -64,406 +66,365 @@ HOST: http://api.apiary.io/
...
+        - resourceName (string) - Resource name
+        - actionName (string) - Action name
+        - exampleName (string) - Example name
+    - duration (number) - Test duration in seconds
+    - result (string) - Result of step execution
+        - Values
+            - `passed`
+            - `failed`
+    - resultData (object) - Data from step execution
+        - request  - [Real HTTP Request](https://www.relishapp.com/apiary/gavel/v/0-1/docs/data-model#http-request)
```
Comment (miiila) [inline ApiaryReportingApi.apib]: I am bit confused by square brackets here (`[]`); it's probably my missing knowledge, but I'd like to clarify before merging.

Comment (miiila) [inline ApiaryReportingApi.apib]: Should be type (`(string)`) included too?

**Comment by honzajavorek** [comment]
Thanks, let me fix the issues before merging.

**Comment by miiila** [comment]
@honzajavorek 👍 I don't want to overflow the discussion with same comments, so please check mentioned issues for duplication.

**Comment by honzajavorek** [comment]
@miiila Updated. All checked for duplicity. (Note: Tests here are not relevant, since these blueprints are tested on Apiary side, not Dredd side. That means this can be considered OK to be merged only once the change which mirrors these changes on Apiary side will pass tests.)

**Comment by miiila** [comment]
Review finished, waiting for finish on Apiary side.

**Pull request merged**